bitnami/mediawiki/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: mediawiki
 sources:
   - https://github.com/bitnami/bitnami-docker-mediawiki
   - http://www.mediawiki.org/
-version: 12.2.11
+version: 12.2.12
```

Comment:
Please, could you bump the minor version instead of the bugfix version (PATCH)?

version: 12.3.0